bfss3/bfss3.go

```diff
@@ -397,7 +409,17 @@ func (i *iterator) fetchNextPage() error {
 		if ok, err := doublestar.Match(i.pattern, name); err != nil {
 			return err
 		} else if ok {
-			i.page = append(i.page, name)
+			i.page = append(
+				i.page,
+				object{
+					name,
+					&bfs.MetaInfo{
```

Comment:
pls add newMetaInfo in this package too

Reply:
I had to add two functions, one each for transforming `s3.HeadObjectOutput` and `s3.Object`, which is what you get in response to calling `ListObjectsV2WithContext`.